src/App/Document.cpp

```diff
@@ -494,7 +495,7 @@ void Document::exportGraphviz(std::ostream& out) const
                 }
                 if(!sgraph) {
                     if(docObj->isDerivedFrom(OriginFeature::getClassTypeId()))
-                        sgraph = GraphList[static_cast<OriginFeature*>(docObj)->getOrigin()];
+                        sgraph = GraphList[dynamic_cast<OriginFeature*>(docObj)->getOrigin()];

```

Comment:
See [forum discussion](https://forum.freecadweb.org/viewtopic.php?p=592717#p592717)

Reply:
Way above my pay grade. Will reverse and leave alone until I get my head around all that.